packages/core/src/2d/sprite/SpriteRenderer.ts

```
@@ -314,17 +314,24 @@ export class SpriteRenderer extends Renderer {
     return this.engine._batcherManager.primitiveChunkManager2D;
   }
 
-  protected override _updateBounds(worldBounds: BoundingBox): void {
-    if (this.sprite) {
-      this._assembler.updatePositions(this);
+  protected override _updateLocalBounds(localBounds: BoundingBox): void {
+    const sprite = this._sprite;
+    if (sprite) {
+      const { width, height } = this;
+      let { x: pivotX, y: pivotY } = sprite.pivot;
+      pivotX = !!this.flipX ? 1 - pivotX : pivotX;
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)